migrations/sqlite/20240809205107_event_source_anchored.up.sql

```diff
@@ -0,0 +1,9 @@
+-- Add up migration script here
+
+-- The CID of the Init Event for the stream
+ALTER TABLE ceramic_one_event ADD COLUMN init_cid BLOB DEFAULT NULL;
+
+-- The ID of the node that sent the event or our own ID if the event came in via API
+ALTER TABLE ceramic_one_event ADD COLUMN informant TEXT DEFAULT NULL;
```

Comment:
Should we create indexes on these two columns? At least on informant if we're querying it now, and we'll likely need one on init_cid if we ever want to actually retrieve events by stream.

Reply:
sounds like we're okay skipping this for now. not sure when we'll query the init_cid and for now, we're willing to seek the highwater mark and then scan from there to filter out by informant